crates/runtime/src/http/routes.rs

```diff
@@ -142,11 +144,11 @@ pub(crate) fn routes(
         .route("/v1/catalogs", get(v1::catalogs::get))
         .route("/v1/datasets", get(v1::datasets::get))
         .route(
```

Comment:
Why this change?

Reply:
axum 0.8 breaking change. See https://tokio.rs/blog/2025-01-01-announcing-axum-0-8-0